services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api.py

```diff
@@ -323,6 +372,20 @@ async def remove_dynamic_sidecar_network(network_name: str) -> bool:
         return False
 
 
+async def remove_dynamic_sidecar_volumes(node_uuid: NodeID) -> bool:
+    async with docker_client() as client:
+        volumes_response = await client.volumes.list(
+            filters={"label": f"uuid={node_uuid}"}
```

Comment:
hmm, I guess the chances are not that big but what if we have 2 deployments on the same swarm. and someone copied a study into the other? this could conflict isn't it?

Reply:
if by copy you mean import/export then that will not happen. The UUIDs are shuffled. Otherwise yes, it will conflict. Also the dynamic-sidecar service will have the same issue. Because it will try to launch a service with the same name.

Reply:
Think I'm missing the point. How would this become an issue with the versioning?

I have not tested this with 2 different swam deployments. In theory if there is no `node_uuid` collision. Or were there any other concerns which I'm missing?